adr/ADR-26.md

```diff
@@ -0,0 +1,236 @@
+# NATS Authorization Callouts
+
+| Metadata | Value          |
+| -------- |----------------|
+| Date     | 2022-11-28     |
+| Author   | @derekcollison |
+| Status   | `Implemented`  |
```

Comment:
no quotes around these - though its such a common mistake i should probably allow it hehe

Reply:
easier yet, remove the quotes from the template... :)